sdk/src/wallet/migration/chrysalis.rs

```diff
@@ -71,6 +71,11 @@ pub async fn migrate_db_chrysalis_to_stardust(
     // `/db` will be appended to the chrysalis storage path, because that's how it was done in the chrysalis wallet
     let chrysalis_storage_path = &(*storage_path_string).join("db");
 
+    if !chrysalis_storage_path.is_dir() {
```

Comment:
Hmm idk, this doesn't seem super robust. We're just relying on the existence of the `/db` part to determine if it's chrysalis or not?

Reply:
Yes because it was always like that and in stardust it's not
Do you also want it to look for the account indexation key or something like that?

Reply:
It just checks if it's empty, if there is a db with random other data it would still overwrite

Reply:
What do you mean random? get_chrysalis_data would only return stuff relevant to chrysalis, right ?

Reply:
Updated